pkg/resource/table/hooks.go

```diff
@@ -150,6 +163,45 @@ func (rm *resourceManager) customUpdateTable(
 	return &resource{ko}, nil
 }
 
+// syncTTL updates a dynamodb table's TimeToLive property.
+func (rm *resourceManager) syncTTL(
+	ctx context.Context,
+	latest *resource,
+	desired *resource,
+) (err error) {
+	rlog := ackrtlog.FromContext(ctx)
+	exit := rlog.Trace("rm.syncTTL")
+	defer func(err error) { exit(err) }(err)
+
+	spec := &svcsdk.TimeToLiveSpecification{}
```

Comment:
nit: because Spec term is already used heavily.
```suggestion
	ttlSpec := &svcsdk.TimeToLiveSpecification{}
```